shared/common-adapters/clickable-box.desktop.js

```diff
@@ -42,7 +42,10 @@ class ClickableBox extends React.Component<Props & {children: any}, {mouseDown:
     let underlay
 
     if (this.state.mouseIn) {
-      const borderRadius = style && style.borderRadius
+      let borderRadius = 0
+      if (style && typeof style === 'object') {
```

Comment:
Not sure if this is the best way to do this

Reply:
we shouldn't be doing this kind of check anymore. Reaching into the style object is super dirty. We do this in only in a couple of places where we kind of 'split' the styles out and apply some to the container we make and some into the actual component. really we should have multiple style objects we expose

Reply:
Okay. I'll try and fix this in a follow-up PR. `Icon` also has a similar thing with `hoverColor` that I'll try to fix.